### ISSUE TITLE
noblacklistfor process child

### ISSUE THREAD

**Issue by liloman** [netblue30/firejail#484]
Related to the issue #475.
```
 noblacklistfor /usr/bin/python3* /usr/bin/youtube-dl
```
I've been checking the source code briefly and I have noticed how the blacklist mechanism works.  (by the way good to have proper comments on the code cause some projects ... :S )

So in order to implement this feature it needs some kind of dynamic approach. 
First I'd like to know if it could be developed and then how.

To the later I reckon several options:

1. Inotify hook on execve to youtube-dl for that profile and unblock python for the child/fork. I'm not sure enough about the inner control for the child.

2. LD_PRELOAD for the profile with a custom filtered execve so if youtube-dl is executed will unveil the block for python.

Like I said I would like to know if It has a green red first.

Cheers

**Comment by netblue30** [comment]
I looked into it, the problem is inotify message from the kernel is asynchronous. By the time you get the message, the program already started. Actually the message has very low priority, you cannot act on it in real time.

I commented out python blacklists in disable-devel.inc, so this will fix your problem for now. I'll add support for globbing in noblacklist command and put real noblacklists in firefox profile.

**Comment by netblue30** [comment]
I'll go with python commented out in disable-devel.inc for the upcoming release. I'll revisit the issue for the release after that.

**Comment by Fred-Barclay** [comment]
Thanks netblue. :smile:

**Comment by liloman** [comment]
That's true. 

I haven't though in that kind of issue. 

Anyway I should better fix actual bugs reported here first than making new features. ;)
